source/content/guides/decoupled/overview/05-manage-settings.md

```diff
@@ -31,13 +31,36 @@ You can change the Site Name, connect or disconnect your Git repository, and del
 
 ### Node.js Version
 
-Pantheon respects the setting in `.nvmrc` when selecting the Node.js version for runtime. Currently supported versions are:
+As of the release of our [V1 build pipeline](/guides/decoupled/overview/v1-pipeline), Front-End Sites runs can run v16, v18, or v20 of Node.js as specified in your `package.json` file.
+
+Non-LTS versions like v17 are not available.
+
+Pantheon inspects the engines field in `package.json` when selecting a specific Node.js version for the build (Gatsby.js and Next.js) and runtime (Next.js) of your Front-End Site. For example, to select Node.js 16 for a given deployment, you should do the following:
+
+```json
```

Comment:
Make it more clear what file this excerpt is from by adding a title: https://docs.pantheon.io/style-guide#file-excerpts 
```suggestion
```json:title=package.json
```